cssselect/parser.py

```diff
@@ -165,14 +165,14 @@ def __repr__(self):
         return "%s[::%s(%r)]" % (
             self.__class__.__name__,
             self.name,
-            [token.value for token in self.arguments],
+            [token.value for token in self.arguments[0]],
```

Comment:
This line rang a bell to me. With no other changes to the class, it means that the `arguments` argument is now expected to be formatted differently, which breaks backward compatibility. Consider the following diff:
```diff
diff --git cssselect/parser.py cssselect/parser.py
index 4351f07..48e7604 100644
--- cssselect/parser.py
+++ cssselect/parser.py
@@ -160,6 +160,7 @@ class FunctionalPseudoElement(object):
     def __init__(self, name, arguments):
         self.name = ascii_lower(name)
         self.arguments = arguments
+        print("arguments:", arguments)
 
     def __repr__(self):
         return "%s[::%s(%r)]" % (
```

Then at the current `master` branch (9edc6c3f5cf558bd99fa9e584c6832fabe24b942):
```python
In [1]: from cssselect import parse

In [2]: p = parse("a::asdf(arg1)")
arguments: [<IDENT 'arg1' at 8>]
```

While at the current `add_support_for_of_s_part_in_nth_child` branch (058b6b5fde4edc9b43fcdfca766b02be1f191ce8):
```python
In [1]: from cssselect import parse

In [2]: p = parse("a::asdf(arg1)")
arguments: ([<IDENT 'arg1' at 8>], None)
```

Could you explain the rationale of this change? Thanks!

Reply:
You're right, the backward compatibility breaks, because I changed signature of method `parse_arguments` which is used both for functions and functional pseudo elements. 
Now I splitted parsing arguments into separate methods, so nothing will break
Nice catch!